libmscore/timesig.cpp

```diff
@@ -299,8 +299,14 @@ void TimeSig::layout()
             ds.clear();
             }
       else {
-            ns = toTimeSigString(_numeratorString.isEmpty()   ? QString::number(_sig.numerator())   : _numeratorString);
-            ds = toTimeSigString(_denominatorString.isEmpty() ? QString::number(_sig.denominator()) : _denominatorString);
+            if (_numeratorString.isEmpty()) {
+                  ns = toTimeSigString(_numeratorString.isEmpty()   ? QString::number(_sig.numerator())   : _numeratorString);
+                  ds = toTimeSigString(_denominatorString.isEmpty() ? QString::number(_sig.denominator()) : _denominatorString);
```

Comment:
I think this broke exist existing scores that had no text specified at all.  Those should simply use the default numerator & denominator values, but here the denominator is left empty.

Reply:
Or, perhaps more commonly (not sure), it seems some scores have the numerator set but not denominator even though the user did not specifically intend that.  This is the case for me, anyhow.